packages/opc-base/src/keyClockAuthProvider/keyClockAuthProvider.ts

```diff
@@ -0,0 +1,151 @@
+import Keycloak from "keycloak-js";
+
+import { Token, UserInfoFn, LocalStorageCreds } from "./types";
+
+/**
+ * Helper class for Red Hat's Internal SSO Authentication
+ */
+export class OpKeyClockAuthProvider {
```

Comment:
Typo in `keycloak`. Same in the file name. Also, refer the keycloak-js library for spelling and capitalization.
```suggestion
export class OpKeycloakAuthProvider {
```

Reply:
Done